### PR TITLE
Test high level items in updated ESDL

### PR DESCRIPTION
**Pull request by KobusVanRooyen**
Created a level test case for checking that the updated (optimized) ESDL is as expected
 @jimrojerTNO please let me know if we need to check some items in more detail or if something important has been missed

### CODE REVIEW

**Comment by FJanssen-TNO** [comment]
Another check that you could add, is if esdl assets are still connected. Basically for every asset, len(asset.port.connectedTo)>=1

File: tests/test_updated_esdl_post_process.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+    def test_updated_esdl(self):
+        """
+        Check that the updated ESDL resulting from the optmizer, is correct by using the PoCTutorial
+        and the Grow_workflow
+
+        Checks:
+        - That the correct number of KPIs have been added
+        - That the correct assets have been removed
+        - Check that all the assets have a state=ENABLED
```
Comment (jimrojerTNO) [inline tests/test_updated_esdl_post_process.py:22]: can we also check:
- The sizes of the pipes and the sizes of source
- can we check that the aggregation count of the asset is set appropiately (I think there might still be a bug there)
- Check that unit of the KPI

Comment (KobusVanRooyen) [reply tests/test_updated_esdl_post_process.py]: Has been added

Comment (jimrojerTNO) [reply tests/test_updated_esdl_post_process.py]: looks good!

**Review by jimrojerTNO — APPROVED** [review]

**Pull request merged**